webapp/src/containers/components/tables/existingConfigTable.tsx

```diff
@@ -85,6 +85,7 @@ const ExistingConfigTable = ({value, onChange}: Props) => {
                     {'Existing Configs'}
                 </div>
                 <div>
+                    {value.length > 0 &&
```

Comment:
```suggestion
                    {value.length &&
```
length can never be negative and `0` is already considered `false` by JS so, checking for just `value.length` means we are checking for `>= 1`